packages/aragon-wrapper/src/utils/index.js

```diff
@@ -11,13 +12,39 @@ export function addressesEqual (first, second) {
   return first === second
 }
 
+export function makeAddressLookupProxy (target) {
+  return new Proxy(target, {
+    get (target, property, receiver) {
```

Comment:
Wouldn't it be simpler to also capture `set` and make sure all addresses are in the same format (lowercased)? Then the getter would only need to convert to lowercase and always use that as the key.

Reply:
Sounded easy on paper, but what an adventure (a5056d8).

Enforcing that the addresses are always lowercase revealed a few more address checks that weren't safe before, but also made it harder to use `dotprop` since `dotprop.get()` doesn't go through the normal getter before checking if an instance exists (https://github.com/sindresorhus/dot-prop/blob/master/index.js#L31; related issue: https://github.com/sindresorhus/dot-prop/issues/46).